python/src/aiconfig/editor/client/src/components/ConfigNameDescription.tsx

```diff
@@ -128,15 +128,21 @@ export default memo(function ConfigNameDescription({
           <Title
             ref={nameDisplayRef}
             onClick={onClickEdit}
-            className={!readOnly ? classes.hoverContainer : undefined}
+            className={
+              !readOnly ? `${classes.hoverContainer} hoverContainer` : undefined
```

Comment:
this line effectively adds two different class names right?

what happens if they both exist? how is the specificity chosen?

Reply:
Correct. The `classes.hoverContainer` is pre-processed by mantine to a dynamic class that serializes the above styles into something like
```
.mantine-<randomid> {
  ... hoverContainer styles above
}
```

And then we add `hoverContainer`as well.

Both styles will be applied but we can ensure global `hoverContainer` has higher specificity since it should be nested under the `div.editorBackground` style as well